specification/vmware/resource-manager/Microsoft.AVS/preview/2021-01-01-preview/examples/Datastores_Create.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "api-version": "2021-01-01-preview",
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "group1",
+    "privateCloudName": "cloud1",
+    "clusterName": "cluster1",
+    "datastoreName": "datastore1",
+    "datastore": {
```

Comment:
This is missing from the spec. I feel like the tooling should flag this as an error.